### ISSUE TITLE
Searchable indentities when creating a new email

### ISSUE THREAD

**Issue by Yoda-BZH** [the-djmaze/snappymail#1805]
**Is your feature request related to a problem? Please describe.**
I have several (50+) identities. When creating a new email, and opening the identity list, the dropdown is quite long.

![image](https://github.com/user-attachments/assets/1b733588-2756-416f-95ee-e7a8813601e9)

**Describe the solution you'd like**
A nice addition would be a search box to filter all identities.

**Describe alternatives you've considered**
none

Thank you

**Comment by the-djmaze** [comment]
Why so many identities?
Do you know you can just edit the `from` field for those you not using a lot?

**Comment by the-djmaze** [comment]
I already thought so 🤣

I do the same, but just edit the from field.
Better would be to have "reply" set the from field where the received message was sent to, then the identities are not needed, don't you think so?

**Comment by bilogic** [comment]
> Better would be to have "reply" set the from field where the received message was sent to, then the identities are not needed, don't you think so?

Absolutely! I always found identities extremely clunky back in K9Mail (which I still use on Android, but mainly only to read).
